memor/session.py

```diff
@@ -257,7 +276,7 @@ def render(self, render_format=RenderFormat.DEFAULT):
         if render_format == RenderFormat.OPENAI:
             result = []
             for message in self._messages:
-                result.extend(message.render(render_format=RenderFormat.OPENAI))
+                result.append(message.render(render_format=RenderFormat.OPENAI))
```

Comment:
Should it be `extend` if we want to have sessions as messages too?

Reply:
fixed in https://github.com/openscilab/memor/pull/49/commits/87b798d30c6d0ff932cbe48e7f7ea8b39ca060d3